.nvmrc

```diff
@@ -0,0 +1 @@
+.node-version
```

Comment:
What's this about?  NVM doesn't have functionality to load version from other files.